server/sql_app/main.py

```diff
@@ -43,9 +63,71 @@ def get_db():
 # operation function and use that session.
 
 
+# To make sure Exceptions aren't silently ignored:
+# https://stackoverflow.com/questions/27297638/when-asyncio-task-gets-stored-after-creation-exceptions-from-task-get-muted/27299160#27299160
+def handle_result(fut):
+    if fut.exception():
+        fut.result()  # This will raise the exception.
+
+
+# Our worker to run tasks
+async def worker(name, queue):
+    """Handle jobs in the priority queue.
+
+    Args:
+        name (string) - name for the asyncio worker task
+        queue (asyncio.PriorityQueue) - priority queue to get jobs from where
+            the returned queue item is a tuple of: (priority (int), job (dict))
+            The job dictionary has keys of:
+                'job' (object) - a function to call
+                'param' (object) - parameters to pass to `job`.
+                'name' (string) - name of job
+                'description' (string) - job description
+                'status' (int) - job status [0 | 1 | 2 | 3]
+                'job_id' (int) - database job ID
+                'db' (database reference) - db session
+    """
+    while True:
+        # Get a "work item" out of the queue.
+        job_priority, job_details = await queue.get()
+        LOGGER.debug(f'job details: {job_details}')
+
+        # I had a try/except here before adding the `add_done_callback` because
+        # there was an exception happening that I was never seeing in the console
+        try:
+            # Update the job status in the DB to "running"
+            job_schema = schemas.JobBase(**{
+                'name': job_details['name'],
+                'description': job_details['description'],
+                'status': 1})
```

Comment:
To make this more readable, what would you say about using a module-level constant, or even switching up the dict so that we have something like one of the below?

 ```
'status': STATUS['running']
```
or
```
'status': STATUS_RUNNING
```